pdm/cli/utils.py

```diff
@@ -565,22 +565,23 @@ def format_resolution_impossible(err: ResolutionImpossible) -> str:
     if len(causes) == 1:
         return (
             "Unable to find a resolution for "
-            f"{termui.green(causes[0].requirement.as_line())}\n"
+            f"{termui.green(causes[0].requirement.identify())}\n"
             "Please make sure the package name is correct."
         )
 
     result = [
-        f"Unable to find a resolution for {termui.green(causes[0].requirement)} "
-        "because the following requirements conflict:"
+        f"Unable to find a resolution for "
+        f"{termui.green(causes[0].requirement.identify())} because of the following "
+        "conflicts:"
     ]
     for req, parent in causes:
         info_lines.add(
             f"  {req.as_line()} (from {repr(parent) if parent else 'project'})"
         )
     result.extend(sorted(info_lines))
     result.append(
-        "To fix this, you could try to loosen the dependency version constraints "
-        "in pyproject.toml."
+        "To fix this, you could loosen the dependency version constraints in "
+        "pyproject.toml. See https://bit.ly/3yJftux for more details."
```

Comment:
(I'm a bit late) I'd prefer seeing a link to the actual documentation page rather than an URL-shortener link, even if it's really long.

Reply:
Fine, already updated